site/detailsOnDemand.tsx

```diff
@@ -17,18 +17,20 @@ declare global {
 }
 
 export async function runDetailsOnDemand() {
-    const details: DetailDictionary = await fetch(
-        `${BAKED_BASE_URL}/dods.json`,
-        {
-            method: "GET",
-            credentials: "same-origin",
-            headers: {
-                Accept: "application/json",
-            },
-        }
-    ).then((res) => res.json())
+    if (!window.details) {
```

Comment:
I guess there's a chance we make this request twice now if the first request doesn't resolve by the time this fires. Not sure how likely that'll be, but I wonder if there's something we can do about it. 🤔

Reply:
If the `Head` component is used on all our pages, we could just remove the second request? We could also await the first request but to make that request blocking is probably too much of a compromise. Thinking about it, if the first request does take forever, we'll probably notice because Grapher's DoDs won't work correctly...